platform-ui-skin/src/main/webapp/skin/less/_responsive.less

```diff
@@ -1105,7 +1096,18 @@
     .quickSearchDisplay .footComment{
         display: none!important;
     }
-    .activeInputSearch .SearchPortletTDContainer{
+	
+    .UITopBarContainerItem.activeInputSearch.SearchPortletTDContainer {

```

Comment:
we shouldn't add css ti parent containers, as explained. The "search box" component should be "auto styled" so that when we place it anywhere or change parent class, it behaves the same way (for maintainability and modularity).